velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -32,15 +32,17 @@ class SimpleAggregatesTest : public WindowTestBase {
             size, [](auto row) -> int32_t { return row % 10; }),
         makeFlatVector<int32_t>(
             size, [](auto row) -> int32_t { return row % 7; }),
-        makeFlatVector<int32_t>(size, [](auto row) -> int32_t { return row; }),
```

Comment:
Presume you made this change since k in (kPreceding or following) needs to be atleast 1 ? If yes, then please add a comment here.